src/main/java/com/redhat/devtools/lsp4ij/actions/LSPShowReformatDialogAction.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.

```

Comment:
where was this code copied from?

Reply:
from ballerina, but I would like expriment a new solution. If it doesn't work, I will add link to this copy/paste code.